xbmc/interfaces/legacy/ListItem.h

```diff
@@ -240,6 +240,7 @@ namespace XBMCAddon
        *     - lyrics        : string (On a dark desert highway...)
        *     - playcount     : integer (2) - number of times this item has been played
        *     - lastplayed    : string (%Y-%m-%d %h:%m:%s = 2009-04-05 23:16:04)
+       *     - mediatype     : string - "song", "album", "artist"
```

Comment:
according to IsValidMediaType(), "music" is also valid. (I have no idea what it should be used for though)